src/spatialdata/models/_utils.py

```diff
@@ -7,15 +7,17 @@
 import geopandas
 import numpy as np
 from dask.dataframe import DataFrame as DaskDataFrame
+from datatree import DataTree
 from geopandas import GeoDataFrame
 from multiscale_spatial_image import MultiscaleSpatialImage
 from shapely.geometry import MultiPolygon, Point, Polygon
 from spatial_image import SpatialImage
+from xarray import DataArray
 
 from spatialdata._logging import logger
 from spatialdata.transformations.transformations import BaseTransformation
 
-SpatialElement = Union[SpatialImage, MultiscaleSpatialImage, GeoDataFrame, DaskDataFrame]
+SpatialElement = Union[SpatialImage, DataArray, MultiscaleSpatialImage, DataTree, GeoDataFrame, DaskDataFrame]
```

Comment:
Here we should remove `SpatialImage` and `MultiscaleSpatialImage` from the `SpatialElements`.

The idea is to use `to_multiscale()` internally, and in `RasterModel.parse()` call internally `to_spatial_image()` and `to_multiscale()` to trigger validation via the `spatial_image` and `multiscale_spatial_image` package, but the user would only be exposed to `DataArray` and `DataTree`.

Reply:
yeah this indeed should still be removed